src/dsp_tools/utils/xml_utils.py

```diff
@@ -103,3 +103,22 @@ def _parse_xml_file(input_file: Union[str, Path]) -> etree._ElementTree[etree._E
     """
     parser = etree.XMLParser(remove_comments=True, remove_pis=True)
     return etree.parse(source=input_file, parser=parser)
+
+
+def remove_namespaces_from_xml(
+    data_xml: Union[etree._ElementTree[etree._Element], etree._Element],
+) -> Union[etree._ElementTree[etree._Element], etree._Element]:
```

Comment:
Not for this PR. But since we discussed changing the XML format - the existence of this function to me suggests, that the new XML format should not use any namespaces.